src/components/admin/sidebar/sidebar.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { IconType } from "react-icons";
-import profileImage from "../../../images/photo-1599566150163-29194dcaad36.avif";
+import profileImage from "../../../assets/images/photo-1599566150163-29194dcaad36.avif";
```

Comment:
Please add baseUrl to ts-config to exclude  relative import .
https://www.typescriptlang.org/docs/handbook/module-resolution.html